From 3438fe135995c11f40e5ae1d91945fb2c6c3c7e6 Mon Sep 17 00:00:00 2001
From: Arnaud Pouliquen <arnaud.pouliquen@foss.st.com>
Date: Fri, 5 Jan 2024 19:02:45 +0100
Subject: [PATCH 0539/1695] pta: stm32mp: fix remoteproc config name

This change fixes the remoteproc PTA configuration name to
CFG_REMOTEPROC_PTA.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git ecdb2810d3fc0716ab139d8da15e75a52388b986 pta: stm32mp: fix remoteproc config name]
Fixes: f6c57ea446db ("pta: stm32mp: add new remoteproc PTA")
Signed-off-by: Arnaud Pouliquen <arnaud.pouliquen@foss.st.com>
Change-Id: I3064af16cb98ff024e36727241025a8c8e4e4405
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/349676
Domain-Review: Arnaud POULIQUEN <arnaud.pouliquen@st.com>
Tested-by: Arnaud POULIQUEN <arnaud.pouliquen@st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Reviewed-by: Arnaud POULIQUEN <arnaud.pouliquen@st.com>
---
 core/pta/stm32mp/sub.mk | 2 +-
 core/pta/sub.mk         | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/core/pta/stm32mp/sub.mk b/core/pta/stm32mp/sub.mk
index f463e32a6..3861457e7 100644
--- a/core/pta/stm32mp/sub.mk
+++ b/core/pta/stm32mp/sub.mk
@@ -1,2 +1,2 @@
 srcs-$(CFG_STM32_BSEC_PTA) += bsec_pta.c
-srcs-$(CFG_RPROC_PTA) += remoteproc_pta.c
\ No newline at end of file
+srcs-$(CFG_REMOTEPROC_PTA) += remoteproc_pta.c
\ No newline at end of file
diff --git a/core/pta/sub.mk b/core/pta/sub.mk
index b7b057147..cf8e86298 100644
--- a/core/pta/sub.mk
+++ b/core/pta/sub.mk
@@ -20,7 +20,7 @@ subdirs-y += stm32mp
 subdirs-y += imx
 subdirs-y += k3
 
-ifeq ($(CFG_RPROC_PTA),y)
+ifeq ($(CFG_REMOTEPROC_PTA),y)
 gensrcs-y += rproc_pub_key
 produce-rproc_pub_key = rproc_pub_key.c
 depends-rproc_pub_key = $(RPROC_SIGN_KEY) scripts/pem_to_pub_c.py
-- 
2.47.2

